packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,56 @@
 
 ## Changelog
 
+### January 23, 2024
+
+#### New Features
+
+- **Deploy Tab Apps to Static Web App**: Azure Static Web Apps, an automatic service for building and deploying full-stack web apps to Azure from a code repository, is now the default solution for deploying Tab-based applications in Teams Toolkit.
```

Comment:
Is there a way to use the old way, or a sample that we can link if the user prefers it?

Reply:
Linked to a sample